tests/foreman/cli/test_organization.py

```diff
@@ -444,60 +305,159 @@ def test_positive_add_user_by_id(self):
         org = make_org()
         user = make_user()
         Org.add_user({
-            'name': org['name'],
+            'id': org['id'],
             'user-id': user['id'],
         })
+        org = Org.info({'id': org['id']})
+        self.assertIn(user['login'], org['users'])
 
     @tier2
     def test_positive_remove_user_by_id(self):
-        """@Test: Check if a User can be removed from an Org
+        """Add an user to organization and remove it by its ID
 
         @Feature: Organization
 
-        @Assert: User is removed from the org
+        @Assert: The user is added then removed from the organization
         """
         org = make_org()
         user = make_user()
         Org.add_user({
-            'name': org['name'],
+            'id': org['id'],
             'user-id': user['id'],
         })
-
         Org.remove_user({
-            'name': org['name'],
+            'id': org['id'],
             'user-id': user['id'],
         })
+        org = Org.info({'id': org['id']})
+        self.assertNotIn(user['login'], org['users'])
 
     @tier2
-    @stubbed()
     def test_positive_remove_user_by_name(self):
-        """@test: Create different types of users then add/remove user
-        by using the organization name
+        """Add an user to organization and remove it by its login and
+        organization name
 
         @feature: Organization
 
         @assert: The user is added then removed from the organization
+        """
+        org = make_org()
+        user = make_user()
+        Org.add_user({
+            'name': org['name'],
+            'user': user['login'],
+        })
+        Org.remove_user({
+            'name': org['name'],
+            'user': user['login'],
+        })
+        org = Org.info({'name': org['name']})
+        self.assertNotIn(user['login'], org['users'])
+
+    @tier2
+    def test_positive_add_admin_user_by_id(self):
+        """Create admin user and add it to organization by user and the
+        organization IDs
+
+        @feature: Organization
 
-        @status: manual
+        @assert: The user is added to the organization
         """
+        org = make_org()
+        user = make_user({'admin': '1'})
+        Org.add_user({
+            'id': org['id'],
+            'user-id': user['id'],
+        })
+        org = Org.info({'id': org['id']})
+        self.assertIn(user['login'], org['users'])
+        self.assertEqual(user['admin'], 'yes')
+
+    @tier2
+    def test_positive_add_admin_user_by_name(self):
+        """Create admin user and add it to organization by login and the
+        organization name
```

Comment:
Test case name needn't explain everything what we do in the test case.  For example, this can be
`Add an admin user to an Org using login name and Org name`